src/libical/icalrecur.c

```diff
@@ -2026,7 +2026,7 @@ icalrecur_iterator *icalrecur_iterator_new(struct icalrecurrencetype rule,
                 impl->orig_data[byrule] = 0;
             } else {
                 icalerror_set_errno(ICAL_MALFORMEDDATA_ERROR);
-                free(impl);
+            icalmemory_free_buffer(impl);
```

Comment:
indentation

Reply:
sorry for that - classical rebasing issues